src/components/Banners/SiteBanner.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { useState, useEffect } from "react";
+import { useQuery } from "@apollo/react-hooks";
+import styled from "@emotion/styled";
+import { uiColors } from "@leafygreen-ui/palette";
+import IconButton from "@leafygreen-ui/icon-button";
+import Icon from "@leafygreen-ui/icon";
+import Cookies from "js-cookie";
+import get from "lodash.get";
+import { GET_SITE_BANNER } from "gql/queries";
+import { SiteBannerQuery } from "gql/generated/types";
+
+const { yellow, blue, green, red } = uiColors;
+export const SiteBanner = () => {
+  const { data, loading } = useQuery<SiteBannerQuery>(GET_SITE_BANNER);
+  const { text, theme } = get(data, "siteBanner", {});
+  const [showBanner, setShowBanner] = useState(false);
+  useEffect(() => {
+    if (text !== "" && Cookies.get(text) === undefined) {
+      setShowBanner(true);
+    }
+  }, [text]);
+  if (loading) return null;
+
+  const hideBanner = () => {
+    // If a user sees a banner and closes it lets set a cookie with the banners text as the key.
+    // The cookie will be auto deleted after a week. This ensures if a new banner with different text is returned we dont accidently hide it
+    setShowBanner(false);
+    Cookies.set(text, "viewed", { expires: 7 });
+  };
+
+  return showBanner ? (
+    <Banner theme={theme}>
+      {text}{" "}
+      <IconButton
+        aria-label="Close Site Banner"
+        variant="light"
+        onClick={hideBanner}
+      >
+        <Icon glyph="X" onClick={hideBanner} />{" "}
+      </IconButton>
+    </Banner>
+  ) : (
+    <BannerPadding />
+  );
+};
+
+type BannerProps = {
+  theme: string;
+};
+
+const Banner = styled.div`
+  transition: max-height 0.3s ease-in-out;
+  align-items: center;
+  ${(props: BannerProps) =>
+    props.theme && `background-color: ${bannerTypeToColor[props.theme]}`};
```

Comment:
I think "" is also a valid option for the theme, which defaults to announcement

Reply:
I dont think thats an option to clear the bannerTheme on the legacy UI admin panel. 
![image](https://user-images.githubusercontent.com/4605522/87049066-8361ca00-c1ca-11ea-906c-1cb475f6840d.png)

But i added support for it anyways just in case.